less/variables.less

```diff
@@ -1,14 +1,84 @@
-
-// Color assignments 
+// Color assignments
 @font-color: @shark;
 @button-bg: @eastern-blue;
-@main-bg:  @white;
+@main-bg: @white;
 @navigation-border: @silver;
 @footer-bg: @sandy-beach;
 
 // Colors
-@white: #FFFFFF;
-@silver: #C0C0C0;
+@white: #ffffff;
+@silver: #c0c0c0;
 @shark: #212529;
-@eastern-blue: #17A2B8;
-@sandy-beach: #FFEBCD;
\ No newline at end of file
+@eastern-blue: #17a2b8;
+@sandy-beach: #ffebcd;
+
+@mobile: ~"(max-width: 500px)";
+
+.flex-column() {
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+@media screen and @mobile {
+  nav {
+    .flex-column();
+    width: 100%;
```

Comment:
There should only be variables in this file, you should not have been writing the bulk of the selectors and Less in here. I know this was an issue we brought up and standup.